Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,250 @@
+  public exponent: 65537
+)
+```
+The .readPublicKeyFrom() function returns a Try of Public/Private key depending on whether the String given can be parsed back to some valid key. If the operation is successful the value of the original key is assigned to the new value in BigInt format.
+
+The user can then save his keys on his local filesystem by using the following ammonite commands:
+
+```scala
+write(wd/"publicKey.pem", RSAKeys.save(pub))
+write(wd/"privateKey.pem", RSAKeys.save(priv))
+```
+This will save the contents of the key in String format within the file found by following the specified path in .pem files.
+
+### Saving a public Key to RDF
+
+A user can also transform his public keys to an RDF. In order to do this one must first import multiple files in order to resolve the required dependencies.
+
+The org.w3 declarations required are:
+
+```scala
+import org.w3.banana.binder
+import org.w3.banana.binder.RecordBinder
+implicit val binder = RecordBinder
+val cert = CertPrefix[Rdf]
+```
+Several java imports from java's security and math library are also required for this process:
+
+```scala
+import java.math.BigInteger
+import java.security.KeyFactory
+import java.security.interfaces.RSAPublicKey
+import java.security.spec.RSAPublicKeySpec
+```
+
+After that the binder and all it's functionalities can be imported:
+
+```scala
+import org.w3.banana.binder._
+import recordBinder._
+```
+
+Finally, after that, the user can create the Cert object that contains the required dependencies to turn a Key into a Pointed Graph
+
+```scala
+object Cert {
+
+          implicit val rsaClassUri = classUrisFor[RSAPublicKey](cert.RSAPublicKey)
+          val factory = KeyFactory.getInstance("RSA")
+          val exponent = property[BigInteger](cert.exponent)
+          val modulus = property[Array[Byte]](cert.modulus)
+
+          implicit val binder: PGBinder[Rdf, RSAPublicKey] =
+          pgb[RSAPublicKey](modulus, exponent)(
+            (m, e) => factory.generatePublic(new RSAPublicKeySpec(new BigInteger(m), e)).asInstanceOf[RSAPublicKey],
+            key => Some((key.getModulus.toByteArray, key.getPublicExponent))
+            ) // withClasses rsaClassUri
+
+      }
+
+defined object Cert
+
+import Cert._
+```
+
+After this all dependencies should be resolved and the user will be able to transform the keys into a Pointed Graph:
+
+```scala
+@ val keyGraph = pub.toPG
+
+keyGraph: PointedGraph[RDF] = org.w3.banana.PointedGraph$$anon$1@7f8fa63d
+```
+One can then retrieve both the pointer and the graph:
+
+```scala
+@ keyPG = keyGraph.graph
```

Comment:
If it is just a graph why call it `keyPG`, which is the extension we use for pointed graph. 
Also missing `val`